From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Thu, 17 Jan 2019 00:31:35 +0100
Subject: tests: increase conform tests timeout

These tests might take much time in some architectures, so we can be more flexible
in accepting later results.

Forwarded: not-needed
---
 clutter/tests/conform/meson.build | 3 ++-
 cogl/tests/conform/meson.build    | 2 +-
 cogl/tests/unit/meson.build       | 1 +
 3 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/clutter/tests/conform/meson.build b/clutter/tests/conform/meson.build
index fc5a617..a85ec45 100644
--- a/clutter/tests/conform/meson.build
+++ b/clutter/tests/conform/meson.build
@@ -76,6 +76,7 @@ foreach test : clutter_conform_tests
   )
 
   test('clutter/conform/@0@'.format(test), test_executable,
-    env: test_env
+    env: test_env,
+    timeout: 60,
   )
 endforeach
diff --git a/cogl/tests/conform/meson.build b/cogl/tests/conform/meson.build
index 43abc0b..4b799f3 100644
--- a/cogl/tests/conform/meson.build
+++ b/cogl/tests/conform/meson.build
@@ -119,5 +119,5 @@ test('cogl/conform', cogl_run_tests,
     cogl_conform_unit_tests
   ],
   is_parallel: false,
-  timeout: 120,
+  timeout: 400,
 )
diff --git a/cogl/tests/unit/meson.build b/cogl/tests/unit/meson.build
index ca852fa..1de3c61 100644
--- a/cogl/tests/unit/meson.build
+++ b/cogl/tests/unit/meson.build
@@ -39,4 +39,5 @@ test('cogl/unit', cogl_run_tests,
     cogl_unit_unit_tests
   ],
   is_parallel: false,
+  timeout: 120,
 )
